inc/az_ulib_interface_manager_1_model.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#ifndef AZ_ULIB_INTERFACE_MANAGER_1_MODEL_H
+#define AZ_ULIB_INTERFACE_MANAGER_1_MODEL_H
+
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_result.h"
+#include "azure/az_core.h"
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdint.h>
+#endif
+
+#include "azure/core/_az_cfg_prefix.h"
+
+/*
+ * interface definition
+ */
+#define INTERFACE_MANAGER_1_INTERFACE_NAME "interface_manager"
+#define INTERFACE_MANAGER_1_INTERFACE_VERSION 1
+
+/*
+ * Define set_default command on interface_manager interface.
```

Comment:
Does this comment describe all the names velow or just `INTERFACE_MANAGER_1_SET_DEFAULT_COMMAND_NAME`?

Reply:
It is for all. But I'll improve it.